domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criteria.java

```diff
@@ -1,232 +1,138 @@
 package org.folio.rest.persist.Criteria;
 
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.apache.commons.io.IOUtils;
 import org.folio.rest.persist.PostgresClient;
-import org.folio.rest.tools.parser.JsonPathParser;
-
-import io.vertx.core.json.JsonObject;
 
 /**
  *
- *         Criteria c = new Criteria();
- *         c.field = "'price' -> 'po_currency' ->> 'value'";
- *         c.operation = "LIKE";
- *         c.value = "USD";
+ * Criteria c = new Criteria(); c.field = "'price' -> 'po_currency' ->>
+ * 'value'"; c.operation = "LIKE"; c.value = "USD";
  *
- *         c.field = "'rush'";
- *         c.operation = "IS TRUE";
- *         c.value = null;
+ * c.field = "'rush'"; c.operation = "IS TRUE"; c.value = null;
```

Comment:
The code example should get reverted to the previous formatting (line wrapping) and be wrapped within some javadoc tags, see https://stackoverflow.com/questions/541920/multiple-line-code-example-in-javadoc-comment

Reply:
Like this?
```
/**
 *
 * Criteria c = new Criteria(); c.field = "'price' -> 'po_currency' ->>
 * 'value'"; c.operation = "LIKE"; c.value = "USD";
 *
 * <code>
 * c.field = "'rush'";
 * c.operation = "IS TRUE";
 * c.value = null;
 * </code>
 */
```

??

Reply:
At least with code tag, the editor leaves it alone on reformat (which is what happened before)

Reply:
This is better, but the rendered javadoc is still a single line with all commands concatenated. Your IDE should show you the rendered javadoc if you move your mouse over "Criteria" of "public class Criteria".